apps/src/templates/SchoolZipSearch.jsx

```diff
@@ -3,23 +3,24 @@ import PropTypes from 'prop-types';
 import i18n from '@cdo/locale';
 import {BodyTwoText} from '@cdo/apps/componentLibrary/typography';
 import style from './school-association.module.scss';
+import classNames from 'classnames';
 import {SimpleDropdown} from '@cdo/apps/componentLibrary/dropdown';
 import SchoolNameInput from '@cdo/apps/templates/SchoolNameInput';
 import Button from '@cdo/apps/templates/Button';
 import analyticsReporter from '@cdo/apps/lib/util/AnalyticsReporter';
 import {EVENTS, PLATFORMS} from '@cdo/apps/lib/util/AnalyticsConstants';
 
-const SELECT_A_SCHOOL = 'selectASchool';
-const CLICK_TO_ADD = 'clickToAdd';
-const NO_SCHOOL_SETTING = 'noSchoolSetting';
+export const SELECT_A_SCHOOL = 'selectASchool';
+export const CLICK_TO_ADD = 'clickToAdd';
+export const NO_SCHOOL_SETTING = 'noSchoolSetting';
```

Comment:
not necessarily something to address in this PR, but I wonder if exporting these consts is indicative of needing a constants file of some kind

Reply:
I made a ticket for this! Good call https://codedotorg.atlassian.net/browse/ACQ-1789